src/_igraph/graphobject.c

```diff
@@ -3656,8 +3656,8 @@ PyObject *igraphmodule_Graph_articulation_points(igraphmodule_GraphObject *self)
  */
 PyObject *igraphmodule_Graph_assortativity_nominal(igraphmodule_GraphObject *self, PyObject *args, PyObject *kwds)
 {
-  static char *kwlist[] = { "types", "directed", NULL };
-  PyObject *types_o = Py_None, *directed = Py_True;
+  static char *kwlist[] = { "types", "directed", "normalized", NULL };
```

Comment:
If you extend `kwlist` with an additional keyword argument, I think you should also update the corresponding `PyArg_ParseTupleAndKeywords` call; the format string should be `O|OO` and you should add `&normalized` as the last argument, after `&directed`. You have done this below for `igraphmodule_Graph_assortativity()` but not here.

Reply:
Just noting that the default must be `normalized=True` on the Python side.

Reply:
thank you